unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Common/ShowHideAnimator/ShowHideAnimator.cs

```diff
@@ -42,6 +39,17 @@ private int visibleParamHash
         }
     }
 
+    private void OnEnable()
+    {
+        if ( hideOnEnable )
+        {
+            Hide(true);
+        }
+    }
```

Comment:
I'm seeing the only changes you applied to this `ShowHideAnimator` class have been about to change the order of some things, right?

Reply:
Mainly, I moved variables which was in-between of public-private methods to be before all methods. Also tried to move events just after properties, and added [UsedImplicitly] attribute. But maybe it was too early and we need to agree on our code organization first...